pkgs/by-name/pl/plant-it-frontend/package.nix

```
@@ -0,0 +1,22 @@
+{
+  lib,
+  flutter326,
+  nix-update-script,
+  plant-it,
+}:
+
+flutter326.buildFlutterApplication {
+  pname = "plant-it-frontend";
+  inherit (plant-it) version;
+  src = "${plant-it.root-src}/frontend";
+
+  targetFlutterPlatform = "web";
+
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = plant-it.meta // {
+    description = "Frontend for Plant It.";
```

```suggestion
    description = "Frontend for Plant It";
```
Please review the requirements for the [meta.description](https://nixos.org/manual/nixpkgs/unstable/#var-meta-description) attribute. No trailing punctuation is permitted.